org/enigma/file/YamlParser.java

```diff
@@ -106,12 +106,14 @@ public String toString()
 		// Implicitly behave as a string
 		public String toString()
 			{
+			if (rawValue == null) return "";
 			return rawValue;
 			}
 
 		public String getValue()
 			{
 			if (escValue == null) escValue = escape(rawValue);
+			if (escValue == null) return "";
```

Comment:
not sure about this one; look for situations in which null checking is happening, here, because unless they're using EmptyOrNull, you've broken them

Reply:
Ok, so that's a little tricky. The only other place it gets called is `getMC(...)`
https://github.com/enigma-dev/lgmplugin/blob/39f981d82d1e0182b7d12ab80bfae2fed6fddfe7/org/enigma/file/YamlParser.java#L71
That normally returns `def`, which is sometimes null, when the key don't exist (which this does not break). I see we would get null in a lot of places this was being called.
https://github.com/enigma-dev/lgmplugin/blob/39f981d82d1e0182b7d12ab80bfae2fed6fddfe7/org/enigma/TargetHandler.java#L289
In some places, like extension name, this would be used without issue.
https://github.com/enigma-dev/lgmplugin/blob/39f981d82d1e0182b7d12ab80bfae2fed6fddfe7/org/enigma/frames/EnigmaSettingsHandler.java#L559
Some other places look like this is not handled so well.
https://github.com/enigma-dev/lgmplugin/blob/39f981d82d1e0182b7d12ab80bfae2fed6fddfe7/org/enigma/frames/EnigmaSettingsHandler.java#L753
What shall I do?